qa/tasks/cephfs/test_admin.py

```diff
@@ -68,6 +68,14 @@ def test_add_data_pool_subdir(self):
         self.mount_a.run_shell(["mkdir", "subdir"])
         self.fs.set_dir_layout(self.mount_a, "subdir", FileLayout(pool=p))
 
+    def test_add_data_pool_non_alphamueric_name_as_subdir(self):
+        """
+        That a new data pool with non-alphanumeric name can be added and used for a sub-directory.
+        """
+        p = self.fs.add_data_pool("I-am-data_pool00.")
+        self.mount_a.run_shell("mkdir subdir")
```

Comment:
This should be ["mkdir", "subdir"] as nautilus doesn't have a lot of enhancements in qa/tasks/cephfs/mount.py
Or we could just cherry-pick,
https://github.com/ceph/ceph/commit/a83ea849d7d
@rishabh-d-dave , what do you think?

Reply:
I cherry-picked a83ea84

Reply:
With the a83ea849d7d2096762a125b8c6dbb8eb9988b2a9, this should be OK now.